input/ignoreWarnings.txt

```diff
@@ -0,0 +1,6 @@
+== Suppressed Messages ==
+
+# Fhirpath is not fhir dependant, hence doesn't have cross version analysis or any dependencies
```

Comment:
FHIRPath...FHIR

Reply:
That's the label/reasoning for the block of warning messages that are supressed.